### DISCUSSION TITLE
Instructions to install?

### DISCUSSION THREAD

**Discussion by one-good-omen** [metasikander/s0str#4] [Q&A] [answered]
May I please have instructions to install?  This is the very server I am seeking to use with Nostr.

Thank you!

**Comment by zerfix** [comment, marked as answer]
The easiest way is to use docker.

1. Install `git`, `docker` & `docker-compose`.
2. Start docker. Usually `systemctl start docker` as root.
3. Pull repo with `git pull https://github.com/metasikander/s0str.git` and enter the folder.
4. Edit `docker-compose.yml` with your own Nostr pubkey.
5. Compile and run s0str with `docker-compose up -d`.

Then you should be able to connect to your s0str server using your server ip at `ws://<ip>:8080`.

**Reply by one-good-omen** [reply]
Thank you!
